src/font/scaled_font.rs

```diff
@@ -118,11 +164,13 @@ impl ScaledFont {
             let mut cluster_count = 0i32;
             let mut cluster_flags = TextClusterFlags::None;
 
+            let text_to_convert = CString::new(text).unwrap();
+
```

Comment:
Ah, I was confused since that didn't compile. I've go a tweaked version coming I hope you'll like.